phoenix-core/src/it/java/org/apache/phoenix/end2end/IndexBuildTimestampIT.java

```diff
@@ -249,7 +249,8 @@ public void testCellTimestamp() throws Exception {
             String selectSql = String.format("SELECT * FROM %s WHERE val = 'abc'", (view ? viewName : dataTableName));
             conn = DriverManager.getConnection(getUrl());
             // assert we are pulling from index table
-            assertExplainPlan(conn, localIndex, selectSql, dataTableName, (view ? "_IDX_" + dataTableName : indexName));
+            assertExplainPlan(conn, localIndex, selectSql, dataTableName, (view && !localIndex ?
+                    "_IDX_" + dataTableName : indexName));
```

Comment:
this doesn't have the indexName(dataTableName) format?

Reply:
we have that format inside assertExplainPlan function here [phoenix-core/src/it/java/org/apache/phoenix/end2end/IndexToolIT.java](https://github.com/apache/phoenix/pull/1737/files/62de58788aed657170cdcfac5fa827cb4230b781#diff-7bc5a84dbd349aa5114d2f1aab874fd3efed28ce5123bcf4ce59bfe6589031ac)